src/Mobs/Rabbit.cpp

```diff
@@ -9,8 +9,27 @@
 
 
 
+static cFastRandom Random;
```

Comment:
Are you sure this is safe? The constructors can be called from plugins, and cFastRandom is not thread safe.